modules/oceanbase/src/main/java/org/testcontainers/oceanbase/OceanBaseCEContainer.java

```diff
@@ -0,0 +1,112 @@
+package org.testcontainers.oceanbase;
+
+import org.apache.commons.lang3.StringUtils;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.utility.DockerImageName;
+
+/**
+ * Testcontainers implementation for OceanBase Community Edition.
+ * <p>
+ * Supported image: {@code oceanbase/oceanbase-ce}
+ * <p>
+ * Exposed ports:
+ * <ul>
+ *     <li>SQL: 2881</li>
+ *     <li>RPC: 2882</li>
+ * </ul>
+ */
+public class OceanBaseCEContainer extends JdbcDatabaseContainer<OceanBaseCEContainer> {
+
+    static final String NAME = "oceanbasece";
+
+    static final String DOCKER_IMAGE_NAME = "oceanbase/oceanbase-ce";
+
+    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse(DOCKER_IMAGE_NAME);
+
+    private static final Integer SQL_PORT = 2881;
+
+    private static final Integer RPC_PORT = 2882;
+
+    private static final String SYSTEM_TENANT_NAME = "sys";
+
+    private static final String DEFAULT_TEST_TENANT_NAME = "test";
+
+    private static final String DEFAULT_USERNAME = "root";
+
+    private static final String DEFAULT_PASSWORD = "";
+
+    private static final String DEFAULT_DATABASE_NAME = "test";
+
+    private String tenantName = DEFAULT_TEST_TENANT_NAME;
+
+    public OceanBaseCEContainer(String dockerImageName) {
+        this(DockerImageName.parse(dockerImageName));
+    }
+
+    public OceanBaseCEContainer(DockerImageName dockerImageName) {
+        super(dockerImageName);
+        dockerImageName.assertCompatibleWith(DEFAULT_IMAGE_NAME);
+
+        addExposedPorts(SQL_PORT, RPC_PORT);
+    }
+
+    @Override
+    public String getDriverClassName() {
+        return OceanBaseJdbcUtils.getDriverClass();
+    }
+
+    @Override
+    public String getJdbcUrl() {
+        return getJdbcUrl(DEFAULT_DATABASE_NAME);
+    }
+
+    public String getJdbcUrl(String databaseName) {
```

Comment:
private?

Reply:
When the container is started with a initial script that changes data in other than the default db, this method would be helpful, so I would suggest to keep it.

Reply:
I think this could be more general for other JDBC implementations. I think for now we can keep it aligned with the rest of them.

Reply:
OK, removed.